src/Products/GenericSetup/PluginIndexes/exportimport.py

```diff
@@ -13,15 +13,14 @@
 """PluginIndexes export / import support.
 """
 
-from zope.component import adapts
-from zope.component import queryMultiAdapter
-
 from Products.PluginIndexes.interfaces import IDateIndex
 from Products.PluginIndexes.interfaces import IDateRangeIndex
 from Products.PluginIndexes.interfaces import IFilteredSet
 from Products.PluginIndexes.interfaces import IPathIndex
 from Products.PluginIndexes.interfaces import IPluggableIndex
 from Products.PluginIndexes.interfaces import ITopicIndex
+from zope.component import adapts
+from zope.component import queryMultiAdapter
```

Comment:
This is broken. That change should not have been made. Looks like the isort policy doesn't work correctly.

Reply:
I forgot to run `isort-apply`, fixed it now.
Maybe we should enforce that the imports are actually sorted by removing the `-` in front of the isort call in `tox.ini`.

Reply:
> Maybe we should enforce that the imports are actually sorted by removing the `-` in front of the isort call in `tox.ini`.

Sure, I wouldn't mind that.